doc/api/stream.md

```diff
@@ -1002,14 +1180,14 @@ class Counter extends Readable {
 }
 ```
 
-#### Example: SimpleProtocol v1 (Sub-optimal)
+#### A Sub-optimal Example: SimpleProtocol (Version 1)
```

Comment:
We should nix this example — it's a bit contrived, and `.unshift` is nearly completely unused.